app/src/main/res/values/ids.xml

```diff
@@ -17,15 +17,37 @@
 
 <resources>
     <!--app_id-->
-    <string name="app_id">"902201000711"</string>
+    <string name="app_id" translatable="false">902201000711</string>
```

Comment:
maybe we should not have the app id and the leaderboard ids on github.
Can we remove them?